src/js/components/chat-input.jsx

```diff
@@ -35,8 +36,8 @@ export class ChatInputComponent extends Component {
     }
 
     onFocus(e) {
-        if (getReadTimestamp() > 0) {
-            updateReadTimestamp(0);
+        if (store.getState().conversation.unreadCount > 0) {
```

Comment:
If you need the component to rerender when that value changes, it needs to go in  the `connect` part at the bottom, if not, it doesn't matter.